### PR TITLE
Drone/0.17

### PR DESCRIPTION
**Pull request by owen-d**
Attempts to fix https://github.com/grafana/loki/pull/4033 by incrementing and rebuilding

### CODE REVIEW

**Comment by owen-d** [comment]
@aknuds1 I've fixed the image issue - it seems (you?) inadvertently pushed up the `0.16.0` tag and it somehow had wrong protoc library versions, which meant all generated protobufs using that image would see diffs. It does, however, seem that the new `golangci-lint` (https://github.com/grafana/loki/pull/4028/files) is much stricter.

File: clients/pkg/promtail/server/server.go [hunk 1]
```diff
@@ -227,7 +227,7 @@ func (s *server) ready(rw http.ResponseWriter, _ *http.Request) {
 
 	rw.WriteHeader(http.StatusOK)
 	if _, err := rw.Write(readinessProbeSuccess); err != nil {
-		level.Error(s.log).Log("msg", "error writing success message", "error", err)
+		_ = level.Error(s.log).Log("msg", "error writing success message", "error", err)
```
Comment (cyriltovena) [inline clients/pkg/promtail/server/server.go:230]: we shouldn't need this everywhere, something must not be working with https://github.com/grafana/loki/blob/main/.golangci.yml#L74-L75

Comment (owen-d) [reply clients/pkg/promtail/server/server.go:230]: Cool, didn't know about that bit @cyriltovena. The new linter definitely felt too strict and I'm not sure why it didn't enforce these exclusions :( (perhaps I made a mistake somewhere?)

We'll either have to merge this PR ro revert https://github.com/grafana/loki/pull/4028 in order to cut a new build image :(

Comment (cyriltovena) [reply clients/pkg/promtail/server/server.go:230]: may be the message has changed since then and the pattern just need correction ?

**Pull request closed without merging**